run.py

```diff
@@ -53,6 +53,8 @@
   --log-level <LEVEL>               Set logging level
   --cluster-name <name>             Name that will be used for cluster creation
   --no-email                        Do not send results email
+  --destroy <PATH>                  Destroy cluster using given path to the
```

Comment:
Just thinking about why to specify path here when we have parameter cluster-path in this PR https://github.com/red-hat-storage/ocs-ci/pull/27/files#diff-4be2078a125fe364f7198675d51bc688R26 . cannot we use this parameter and --destroy will be just true or false depends if defined?

Reply:
`--cluster-path` doesn't exist on master yet which is why I didn't use it :) We can change this to have `--destroy` set a boolean and pair it up with `--cluster-path` if that is better.